pre_commit_hooks/shellcheck_run_steps.py

```diff
@@ -109,6 +109,12 @@ def main(argv: Sequence[str] | None = None) -> int:
             "w",
             delete_on_close=False,
         ) as compiled_out:
+            with open(filename) as precompiled_in:
+                melange_cfg = yaml.load(precompiled_in)
+                architectures = melange_cfg["package"].get("target-architecture", [])
+                if not architectures:
+                    architectures = ["x86_64"]
```

Comment:
Could you use `architectures = melange_cfg["package"].get("target-architecture", ["x86_64"])` instead of the separate `if not architectures...` ?